### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by naveenreddy0**
added buttons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,6 +20,16 @@
 
     </head>
     <body >
+	    
+	    
+	    <div class="lang_switcher">
```
Comment (naveenreddy0) [inline index.html:25]: The idea of adding buttons is good.

File: index.html [hunk 2]
```diff
@@ -20,6 +20,16 @@
...
+        <center>
+            <button type="submit" name="lang" value="Irish" onclick="Irish()">Gaeilge</button>
+            <button type="submit" name="lang" value="English" onclick="English()">English</button>
+        </center>
+    </div>
```
Comment (naveenreddy0) [inline index.html:30]: Adding some effects for buttons makes the webpage look good.

File: index.html [hunk 3]
```diff
@@ -20,6 +20,16 @@
...
+	    
+	    
         <div class="container">
             <div class="imagebg"></div>
             <div class="row " style="margin-top: 50px">
```
Comment (naveenreddy0) [inline index.html:35]: make sure that your buttons work for that you should add javascript to change the language on a webpage.

File: index.html (hunk 1 above)
Comment (gantlavamshikrishnareddy) [inline index.html:25]: the idea of adding a button is good.

File: index.html (hunk 2 above)
Comment (gantlavamshikrishnareddy) [inline index.html:30]: Adding some effects for buttons makes the webpage look good.

File: index.html (hunk 3 above)
Comment (gantlavamshikrishnareddy) [inline index.html:35]: make sure that your buttons work for that you should add javascript to change the language on a webpage.

**Pull request closed without merging**